content/en/cloud_cost_management/setup/aws.md

```diff
@@ -140,27 +171,21 @@ Attach the new S3 policy to the Datadog integration role.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
 
+{{% /tab %}}
```

Comment:
```suggestion
[1]: https://docs.aws.amazon.com/cur/latest/userguide/dataexports-create-legacy.html
[4]: https://docs.aws.amazon.com/cur/latest/userguide/dataexports-view.html
[10]: https://docs.aws.amazon.com/cur/latest/userguide/enabling-split-cost-allocation-data.html
{{% /tab %}}
```

Reply:
This suggestion should get the links to work as expected, but I recommend updating the in-tab reference links to be unique (perhaps beginning with `200`) for ease of maintenance.